compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -621,6 +644,63 @@ def _event_loop(self):
                     f"    args: {msg_a}\n"
                     f"  kwargs: {msg_kw}{identity_for_log}"
                 )
+                self.cmd_send_failure(command_kwargs, user_ident=identity_for_log)
+
+    def cmd_send_failure(
```

Comment:
As discussed out-of-band, to maintain consistency, I think we should only prefix method names with `cmd_` when they are for commands sent from the web service, not vice-versa.

Reply:
Good point.  Have gone with `send_failure_notice`.